iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -61,14 +61,11 @@ PortManager::PortManager(RouDiMemoryInterface* roudiMemoryInterface) noexcept
 
     popo::PublisherOptions options;
     options.historyCapacity = 1;
+    options.nodeName = INTROSPECTION_SERVICE_ID;
```

Comment:
This is something like a built-in node the introspection publisher port belongs to? `INTROSPECTION_SERVICE_ID` is maybe a bad name. Would this be `INTROSPECTION_NODE`?